src/components/Kurlsh.js

```diff
@@ -591,8 +591,19 @@ class Kurlsh extends React.Component {
 
   getLabel = name => ({ version }) => {
     if (version === "latest") {
-      const latest = this.state.versions[name][1];
-      version = `latest (${latest.version})`;
+      if (name === "kubernetes") {
+        const latest = this.state.versions[name][2]; // for k8s, the first version is a ".x" version
+        version = `latest (${latest.version})`;
+      } else {
+        const latest = this.state.versions[name][1];
+        version = `latest (${latest.version})`;
+      }
+    } else if (version.endsWith(".x")) {
```

Comment:
does this also needs a check is name kubernetes?

Reply:
I'll put it in for now